onnxruntime/python/tools/transformers/models/whisper/whisper_chain.py

```diff
@@ -66,8 +57,8 @@ def chain_model(args):
     num_return_sequences = helper.make_tensor_value_info("num_return_sequences", TensorProto.INT32, [1])
     length_penalty = helper.make_tensor_value_info("length_penalty", TensorProto.FLOAT, [1])
     repetition_penalty = helper.make_tensor_value_info("repetition_penalty", TensorProto.FLOAT, [1])
-    attention_mask = helper.make_tensor_value_info(
-        "attention_mask", TensorProto.INT32, ["batch_size", "feature_size", "sequence_length"]
+    decoder_input_ids = helper.make_tensor_value_info(
```

Comment:
`decoder_input_ids` is new optional input - perhaps make the inclusion of this input behind a flag in the export script? E.g. 

`if args.use_decoder_input_ids:
    decoder_input_ids = helper...`

Reply:
Fixed